### PR TITLE
updates for helm

### PR DESCRIPTION
**Pull request by jmazzitelli**
goes with:

* https://github.com/kiali/kiali/issues/195
* https://github.com/kiali/kiali-operator/pull/93

### CODE REVIEW

File: content/documentation/edge/installation-guide/index.adoc [hunk 1]
```diff
@@ -83,7 +83,7 @@ If you are installing on OpenShift, you must grant the `cluster-admin` role to t
 
 == Install Kiali via Istio or Maistra
```
Comment (jshaughn) [inline content/documentation/edge/installation-guide/index.adoc:84]: While you're in the install guide you may as well update the Version Compatibility section above.   Istio 1.7+ can set Kiali 1.22+ with a note about the installation change.

File: content/documentation/edge/installation-guide/index.adoc [hunk 2]
```diff
@@ -83,7 +83,7 @@ If you are installing on OpenShift, you must grant the `cluster-admin` role to t
...
 
-icon:lightbulb[size=1x]{nbsp} Both Istio and Maistra include Kiali distributions.
+icon:bullhorn[size=1x]{nbsp} Starting with Istio 1.7, Kiali is no longer shipped with the istioctl installer. If you are using an  Istio implementation of version 1.7 or higher, this section is not valid for you.
```
Comment (jshaughn) [inline content/documentation/edge/installation-guide/index.adoc:86]: You should tell them what IS valid, maybe link them there.

File: content/documentation/edge/installation-guide/index.adoc [hunk 3]
```diff
@@ -99,35 +99,35 @@ The version of Kiali packaged with Istio or Maistra may not contain the most rec
...
 
-Below are three options for installing the Kiali Operator. If you're replacing an existing Kiali Operator you will be prompted for confirmation. Depending on the installation option the operator may then install Kiali. If you're replacing an existing Kiali you will be prompted for confirmation. For new Kiali installations, you will be prompted for the authentication strategy. See link:#_login_options[Login Options] for more on the `login`, `anonymous`, `ldap`, `openshift`, `token` and `openid` authentication strategies. Depending on the chosen strategy, the installation may prompt for additional information.
+The next few sections describe the different installation methods available for installing the Kiali Operator. Depending on the installation method you use, the operator may then install Kiali. Before you install Kiali, you will want to know what authentication strategy Kiali should use (this determines how users are to log into Kiali). See link:#_login_options[Login Options] for more on the `login`, `anonymous`, `ldap`, `openshift`, `token` and `openid` authentication strategies.
```
Comment (jshaughn) [inline content/documentation/edge/installation-guide/index.adoc:102]: what authentication strategy --> which authentication strategy

File: content/documentation/edge/installation-guide/index.adoc [hunk 4]
```diff
@@ -99,35 +99,35 @@ The version of Kiali packaged with Istio or Maistra may not contain the most rec
...
-When the Kiali CR is created it triggers the operator to then install Kiali. Kiali is able to access existing namespaces, but it required a CR update to access future namespaces. See link:#_namespace_management[Namespace Management] for more information. Run the install script via this command:
+This option installs the Kiali Operator and the Kiali CR with default options. If you are running with an https://github.com/istio/istio/releases[upstream Istio release], you will probably want to pass in _--accessible-namespaces="**"_ since that is the typical way Kiali is expected to run within an upstream Istio environment.
 
 [source,bash]
 ----
-  bash <(curl -L https://kiali.io/getLatestKialiOperator) --accessible-namespaces '**'
+  bash <(curl -L https://kiali.io/getLatestKialiOperator) --accessible-namespaces="**"
 ----
 
+In this case, Kiali can access currently existing namespaces as well as future namespaces that may be created later. See link:#_namespace_management"[Namespace Management] for more information if you want to change that behavior.
```
Comment (jshaughn) [inline content/documentation/edge/installation-guide/index.adoc:116]: In this case...,  -->  This option gives Kiali access to existing namespaces as well as namespaces created later.

File: content/documentation/edge/installation-guide/index.adoc [hunk 5]
```diff
@@ -153,8 +153,30 @@ It is not usually necessary to explicitly execute the link:https://github.com/ki
...
 ----
 
-icon:bullhorn[size=1x]{nbsp} The install script requires `envsubst` installed and in your PATH; you can get it via the GNU `gettext` package.
+==== OperatorHub
+
+For production environments that have OperatorHub installed (such as OpenShift), you will want to install Kiali Operator using OperatorHub. Simply go to the OperatorHub console and install Kiali Operator. At that point, you can link:#_create_or_edit_the_kiali_cr[create the Kiali CR] to install Kiali.
+
+==== Helm Chart
+
+For production environments that do not have OperatorHub, you will most likely want to use link:https://kiali.org/kiali-operator/charts/index.yaml[the Helm Chart] rather than the install script. In fact, the install script uses the Helm Chart to perform the installation.
```
Comment (jshaughn) [inline content/documentation/edge/installation-guide/index.adoc:162]: maybe the link text should be [the Kiali Operator Helm Chart]

File: content/documentation/edge/quick-start/index.adoc [hunk 6]
```diff
@@ -15,27 +15,64 @@ draft: false
 :imagesdir: /images/quickinstall/
 :sectlinks:
 
-icon:lightbulb[size=1x]{nbsp} This page is intended for users installing Kiali for the first time, not recomended for production environments. In case of custom installations, visit the link:{{< ref "/documentation/edge/installation-guide" >}}[installation guide].
+icon:lightbulb[size=1x]{nbsp} This page is intended for users installing Kiali for the first time, not recomended for custom production environments. In case of custom installations, visit the link:{{< ref "/documentation/edge/installation-guide" >}}[installation guide].
```
Comment (jshaughn) [inline content/documentation/edge/quick-start/index.adoc:18]: typo: recommended

File: content/documentation/edge/quick-start/index.adoc [hunk 7]
```diff
@@ -46,9 +83,9 @@ Once you have the `istioctl` in your PATH, execute the following:
 istioctl install --set profile=demo
 ----
 
-=== Access to the UI
+==== Access to the UI
 
-The Istio client has an easy method to access to all of its UIs. To get access to the Kiali UI:
+The Istio `istioctl` client has an easy method to access to all of its UIs. To get access to the Kiali UI:
```
Comment (jshaughn) [inline content/documentation/edge/quick-start/index.adoc:88]: Is this still true with the addons change?

File: content/documentation/edge/quick-start/index.adoc [hunk 8]
```diff
@@ -57,16 +94,30 @@ istioctl dashboard kiali
...
 $ oc patch clusterrole kiali -p '[{"op":"add", "path":"/rules/-", "value":{"apiGroups":["apps.openshift.io"], "resources":["deploymentconfigs"],"verbs": ["get", "list", "watch"]}}]' --type json
 $ oc patch clusterrole kiali -p '[{"op":"add", "path":"/rules/-", "value":{"apiGroups":["project.openshift.io"], "resources":["projects"],"verbs": ["get"]}}]' --type json
 $ oc patch clusterrole kiali -p '[{"op":"add", "path":"/rules/-", "value":{"apiGroups":["route.openshift.io"], "resources":["routes"],"verbs": ["get"]}}]' --type json
 ----
 
+== Open the UI
+
+Once the Kiali Server has been installed and has successfully started, you can access the UI. For more details, check the link:{{< ref "/documentation/edge/faq/general#how-do-i-access-kiai" >}}[FAQ: How do I access Kiali UI?]
+
+The Istio `istioctl` client has an easy method to access the Kiali UI:
```
Comment (jshaughn) [inline content/documentation/edge/quick-start/index.adoc:111]: Again just asking that this is still gonna be there...

File: content/documentation/edge/quick-start/index.adoc [hunk 9]
```diff
@@ -57,16 +94,30 @@ istioctl dashboard kiali
...
+
+[source,bash]
+----
+istioctl dashboard kiali
+----
+
+icon:bullhorn[size=1x]{nbsp} The credentials you use on the login screen depend on the authentication strategy that was configured for Kiali. See link:{{< ref "/documentation/edge/installation-guide#_login_options" >}}[Login Options] for more details.
```
Comment (jshaughn) [inline content/documentation/edge/quick-start/index.adoc:118]: authentication strategy that was configured --> authentication strategy configured

File: content/documentation/edge/installation-guide/index.adoc (hunk 1 above)
Comment (jmazzitelli) [reply content/documentation/edge/installation-guide/index.adoc:84]: I'm adding this:
```
| 1.7.0+
| 1.22.0+
| Istio 1.7 istioctl will no longer install Kiali. Use the Istio samples/addons all-in-one yaml or the Kiali Helm Chart.
```

File: content/documentation/edge/installation-guide/index.adoc (hunk 2 above)
Comment (jmazzitelli) [reply content/documentation/edge/installation-guide/index.adoc:86]: Changing to: `If you are using an  Istio implementation of version 1.7 or higher, skip this section and go on to the next.`

File: content/documentation/edge/installation-guide/index.adoc (hunk 3 above)
Comment (jmazzitelli) [reply content/documentation/edge/installation-guide/index.adoc:102]: will be done.

File: content/documentation/edge/installation-guide/index.adoc (hunk 4 above)
Comment (jmazzitelli) [reply content/documentation/edge/installation-guide/index.adoc:116]: will be done

File: content/documentation/edge/installation-guide/index.adoc (hunk 5 above)
Comment (jmazzitelli) [reply content/documentation/edge/installation-guide/index.adoc:162]: will be done

File: content/documentation/edge/quick-start/index.adoc (hunk 6 above)
Comment (jmazzitelli) [reply content/documentation/edge/quick-start/index.adoc:18]: will be done

File: content/documentation/edge/quick-start/index.adoc (hunk 7 above)
Comment (jmazzitelli) [reply content/documentation/edge/quick-start/index.adoc:88]: I think it is. All that does is create a port-forward. If you have kiali installed, however you installed it, that should work.

File: content/documentation/edge/quick-start/index.adoc (hunk 8 above)
Comment (jmazzitelli) [reply content/documentation/edge/quick-start/index.adoc:111]: Yes, I think so.

File: content/documentation/edge/quick-start/index.adoc (hunk 9 above)
Comment (jmazzitelli) [reply content/documentation/edge/quick-start/index.adoc:118]: will be done

**Pull request merged**